observer/lib/observer.js

```diff
@@ -9,9 +9,9 @@ import * as Sentry from '@sentry/node'
  */
 export const observeTransferEvents = async (pgPoolStats, ieContract, provider) => {
   const { rows } = await pgPoolStats.query(
-    'SELECT MAX(last_checked_block) FROM daily_reward_transfers'
+    'SELECT MAX(last_checked_block) AS last_checked_block FROM daily_reward_transfers'
   )
-  let queryFromBlock = rows[0].last_checked_block
+  let queryFromBlock = rows[0].last_checked_block + 1
```

Comment:
I think there is bug lurking here. 

In the initial run against an empty database, `last_checked_block` will be `undefined` and `queryFromBlock` will be set to `NaN`.

```
❯ node          
Welcome to Node.js v20.14.0.
Type ".help" for more information.
> undefined + 1
NaN
```

Reply:
You're right!

Reply:
We check the truthy value of the queryFromBlock directly after this, so it ends up being reset to the -1900 block